src/main/java/me/lokka30/littlethings/modules/MobSilentModule.java

```diff
@@ -0,0 +1,101 @@
+package me.lokka30.littlethings.modules;
+
+import me.lokka30.littlethings.LittleModule;
+import me.lokka30.littlethings.LittleThings;
+import me.lokka30.microlib.other.VersionUtils;
+import org.bukkit.Bukkit;
+import org.bukkit.configuration.file.YamlConfiguration;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntitySpawnEvent;
+
+import java.io.File;
+
+public class MobSilentModule implements LittleModule {
+    public boolean isEnabled;
+    private LittleThings instance;
+    private YamlConfiguration moduleConfig;
+
+    @Override
+    public boolean isEnabled() {
+        return isEnabled;
+    }
+
+    @Override
+    public String getName() {
+        return "MobSilent";
+    }
+
+    @Override
+    public int getInstalledConfigVersion() {
+        return moduleConfig.getInt("file-version");
+    }
+
+    @Override
+    public int getLatestConfigVersion() {
+        return 1;
+    }
+
+    @Override
+    public void loadConfig() {
+        File moduleConfigFile = instance.getModuleConfigFile(getName());
+
+        //Check if the file exists, create it if necessary.
+        if (!moduleConfigFile.exists()) {
+            instance.saveModuleConfigFile(getName());
+        }
+
+        moduleConfig = YamlConfiguration.loadConfiguration(moduleConfigFile);
+        moduleConfig.options().copyDefaults(true);
+
+        isEnabled = moduleConfig.getBoolean("enabled");
+    }
+
+    @Override
+    public void loadModule() {
+        instance = LittleThings.getInstance();
+        loadConfig();
+        if (isEnabled) {
+            if (VersionUtils.isOneNine()) {
+                Bukkit.getPluginManager().registerEvents(new Listeners(), LittleThings.getInstance());
+            } else {
+                instance.logger.error("MobSilent module is enabled but your server is not MC 1.9+. Please disable the module as it will have no effect on your server.");
```

Comment:
Note to self: replace start of string with `"The &bMobSilent&7 module is enabled .......`